scalecube-configuration/src/main/java/io/scalecube/configuration/repository/couchbase/CouchbaseRepository.java

```diff
@@ -79,37 +97,94 @@ public CouchbaseRepository(AsyncBucket bucket) {
   }
 
   @Override
-  public Flux<Document> readList(String tenant, String repository) {
+  public Flux<Document> readAll(String tenant, String repository, Integer version) {
+    return Flux.from(
+            RxReactiveStreams.toPublisher(
+                bucket.query(
+                    ViewQuery.from("keys", "by_keys").key(tenant + DELIMITER + repository))))
+        .flatMap(asyncViewResult -> RxReactiveStreams.toPublisher(asyncViewResult.rows()))
+        .flatMap(
+            asyncViewRow ->
+                read(tenant, repository, asyncViewRow.id().split("::")[INDEX_OF_KEY], version))
```

Comment:
Please document this convention of `something::another::whatever` because it's not quite readable.
Maybe you should implement some methods (I guess one for create a key and another to extract one) and write them near each other - to make this more readable.

Reply:
Yes, it can be option, but there is only long record `asyncViewRow.id().split("::")[INDEX_OF_KEY]` and to create method for this redundant. So the user (developer) may see pop-up java doc for `read` method to understand params.

Reply:
Maybe create s constant with `"::"` and add some javadoc on it.
Moreover, you have already this constant - see `io.scalecube.configuration.repository.couchbase.CouchbaseRepository#DELIMITER`

Reply:
This line `asyncViewRow.id().split("::")[INDEX_OF_KEY]` is not meaningful and therefore not readable. Please change it. 
The lambda parameter `asyncViewRow`'s name has no meaning. Please rename to something meaningful.

Reply:
I added delimiter for "::", but to change asyncViewRow isn't necessary, it's just simple row of indexed view (async row)